docs/reference/configuration/istio-sidecar-vs-ambient.md

```diff
@@ -0,0 +1,91 @@
+
+In Ambient Mesh mode, Istio uses node-level proxies (ztunnel) and optional waypoint proxies to provide service mesh functionality. ztunnel runs on each node and intercepts traffic at the node level. Waypoint proxies are deployed alongside applications that require more fine-grained control or advanced features.
+
+**Pros:**
+
+*   **Lower Resource Overhead:** Reduces resource consumption as proxies are shared across multiple pods on a node.
+*   **Simplified Deployments:** Simplifies pod deployments as no sidecar injection is required by default.
+*   **Less Disruptive Upgrades:** Upgrades to ztunnel are less disruptive as they don't require pod restarts.
+
+**Cons:**
+
+*   **Migration:** If migrating from Sidecar to Ambient, planning must be done ahead of time to ensure a smooth transition. However, he UDS Operator automatically handles Istio configuration changes for you, making the transition an easier process.
+*   **Less Mature than Sidecar:** Ambient is a newer feature than Sidecar, this not all bugs have been discovered yet.
+*   **Traffic Shift:** With the introduction of Ambient, Layer 7 Authorization features are only available via opting into an optional Envoy proxy via the Waypoint resource.
+
+**When to Use Ambient Mesh Mode:**
+
+*   When you want to minimize resource consumption.
+*   When you want to simplify application deployments.
+*   For most applications that don't require very fine-grained policy enforcement or features not yet supported in Ambient Mesh.
+*   For help deciding, [this](https://blog.howardjohn.info/posts/opinionated-istio/#ambient-mode) article is a recommended read.
+
+**Configuration:**
+
+You can use ambient mode by setting `spec.network.serviceMesh.mode: ambient` in your [UDS Package](/reference/configuration/uds-operator/package/). When `spec.network.serviceMesh.mode` is not configured, the UDS Operator will default your Package to Sidecar mode.
+
+## Choosing the Right Mode
+
+The choice between Sidecar and Ambient mode depends on your specific requirements and priorities:
+
+*   **Resource Optimization:** If minimizing resource consumption is a top priority, Ambient Mesh is the better choice.
+*   **Security and Control:** If you require very strong isolation and fine-grained control, Sidecar mode might be necessary. However, Waypoint proxies in Ambient Mesh can provide similar levels of control when needed.
+*   **Simplicity:** If you want to simplify application deployments and reduce operational overhead, Ambient Mesh is a good option.
+*   **Feature Requirements:** Evaluate whether all the features you need are supported in Ambient Mesh. If not, you'll need to use Sidecar mode or consider using Waypoint Proxies with ambient mode when possible.
+
+Sidecar mode remains a valuable option for specific use cases where its strengths are required. For more resource aware deployments, Ambient offers a more streamlined approach that saves CPU, Memory, and Networking overhead as your environment scales.
```

Comment:
I don't know if its worth adding one more sentence here explicitly restating that you can make this choice per application/namespace with Package CR? Basically just to note that using both in the same cluster is totally fine and can optimize around what each app needs?